packages/@react-aria/button/docs/useButton.mdx

```diff
@@ -19,6 +19,10 @@ import {useButton} from '@react-aria/button';
 import {useRef} from 'react';
 ```
 
+---
+category: Buttons
```

Comment:
what other data do you think might be added here?
any ways to know what the available categories are? i can see myself just writing slightly wrong ones pretty easily (misspellings being minor, but completely different name since some of them are a bit arbitrary and i've never been able to navigate them easily in spectrums website)

Reply:
project wide search for `category: `? 😄